python/cudf/cudf/core/column/column.py

```diff
@@ -1513,6 +1513,7 @@ def column_empty(
                         * cudf.dtype(libcudf.types.size_type_dtype).itemsize
                     )
                 ),
+                size=None,
```

Comment:
In the long run, this makes me wonder whether we should either always require a non-None size or if we should always calculate it using the logic we have. The latter would ensure internal consistency, while the former would be more consistent with libcudf.

Reply:
I would be fine with the former since we'll be moving more toward pylibcudf (with their column constructors maybe?)